docs/PRODUCTION.md

```diff
@@ -369,3 +369,129 @@ scrape_configs:
 Note: VictoriaMetrics generally performs better than Prometheus.
 
 <!-- TODO: sample dashboards and JSON. -->
+
+## Automatic database backups
+
+You can use [Litestream](https://litestream.io/) for real-time database backups.
+
+### Installation
+
+1. Install Litestream.
+
+    ```bash
+    wget -qO- https://github.com/benbjohnson/litestream/releases/download/v0.3.9/litestream-v0.3.9-linux-amd64-static.tar.gz | sudo tar xzvf - --no-same-owner -C /usr/local/bin litestream
+    ```
+
+2. Create a template systemd service and configuration directory.
+
+    ```bash
+    # note: we restrict the permissions since the configs usually contain credentials
+    sudo mkdir --mode=0700 /etc/litestream
+    sudo nano /etc/systemd/system/litestream@.service
+    ```
+
+    ```ini
+    [Unit]
+    Description=Litestream (%I)
+
+    [Service]
+    User=root
+    Group=root
+
+    Restart=always
+    ExecStart=/usr/local/bin/litestream replicate -config /etc/litestream/%I.yml
+
+    ProtectClock=yes
+    ProtectKernelTunables=yes
+    ProtectKernelModules=yes
+    ProtectKernelLogs=yes
+    ProtectControlGroups=yes
+    PrivateTmp=yes
+    PrivateDevices=yes
+    PrivateMounts=yes
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+
+3. Create the configuration for Atlas.
+
+    ```bash
+    sudo nano /etc/litestream/atlas.yml
+    ```
+
+    ```yml
+    dbs:
+    - path: /var/lib/atlas/accounts.db
+      replicas:
+      - url: sftp://USER:PASSWORD@SERVER:PORT/litestream/accounts
+        retention: 336h
```

Comment:
I'd add a comment with `14d`. At least to me `336h` is less common then e.g. `300s` so it feels a bit more like a _magic value_. ^^

Reply:
Done.